### PR TITLE
[pyth-evm-price-pusher] Add price configuration file

### PR DESCRIPTION
**Pull request by ali-bahjati**
I am planning to make trigger parameters optional and some other changes for the config. I didn't do it to keep the changeset here smaller (and the same as old logic) and will do them on a separate PR.

### CODE REVIEW

File: pyth-evm-price-pusher/README.md [hunk 1]
```diff
@@ -22,12 +22,23 @@ This approach is useful for protocols that already depend on regular push update
...
+- Time difference: The on-chain price is older than `time_difference` seconds
   from the latest Pyth price.
-- Price deviation: The latest Pyth price feed has changed more than `price-deviation` percent
+- Price deviation: The latest Pyth price feed has changed more than `price_deviation` percent
   from the on-chain price feed price.
 - Confidence ratio: The latest Pyth price feed has confidence to price ratio of more than
-  `confidence-ratio`.
+  `confidence_ratio`.
+
+Parameters above are configured per price feed in a price configuration YAML file. The structure looks like this:
```
Comment (jayantk) [inline pyth-evm-price-pusher/README.md:32]: "The parameters above ..."

File: pyth-evm-price-pusher/README.md [hunk 2]
```diff
@@ -80,6 +86,8 @@ For example, to push `BTC/USD` and `BNB/USD` prices on BNB testnet, run the foll
 ```sh
 npm run start -- --evm-endpoint "https://data-seed-prebsc-1-s1.binance.org:8545" --mnemonic-file "path/to/mnemonic.txt" \
   --pyth-contract bnb_testnet --price-endpoint https://prices.testnet.pyth.network \
-  --price-ids "f9c0172ba10dfa4d19088d94f5bf61d3b54d5bd7483a322a982e1373ee8ea31b" "ecf553770d9b10965f8fb64771e93f5690a182edc32be4a3236e0caaa6e0581a" \
-  --time-difference 60 --price-deviation 0.5 --confidence-ratio 1
+  --price-config-file "price-config.testnet.sample.yaml"
 ```
+
+[`price-config.testnet.sample.yaml`](./price-config.testnet.sample.yaml) contains configuration for `BTC/USD`
```
Comment (jayantk) [inline pyth-evm-price-pusher/README.md:92]: 👍 for providing a sample file

File: pyth-evm-price-pusher/price-config.testnet.sample.yaml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+- alias: BTC/USD
```
Comment (jayantk) [inline pyth-evm-price-pusher/price-config.testnet.sample.yaml:1]: I guess we don't send the symbol names so we can't get these names from the price service?

File: pyth-evm-price-pusher/src/evm-price-listener.ts [hunk 4]
```diff
@@ -86,7 +91,9 @@ export class EvmPriceListener implements PriceListener {
 
     const priceId = removeLeading0x(event.returnValues.id);
     console.log(
-      `Received a new Evm PriceFeedUpdate event for price feed with id ${priceId}`
+      `Received a new Evm PriceFeedUpdate event for price feed with id ${priceId} (${this.priceIdToAlias.get(
```
Comment (jayantk) [inline pyth-evm-price-pusher/src/evm-price-listener.ts:94]: nitpick: i would swap the alias and the ID here. the alias is more human readable, so put that first and the id in parens after.

**Review by jayantk — APPROVED** [review]
nice!

File: pyth-evm-price-pusher/price-config.testnet.sample.yaml (hunk 3 above)
Comment (ali-bahjati) [reply pyth-evm-price-pusher/price-config.testnet.sample.yaml:1]: yeah we don't.. in general I think it's good to add symbol name to PriceFeed data structure somehow, what do you think? 

The problem is that on solana if we want that we should add it to the price account.

File: pyth-evm-price-pusher/src/evm-price-listener.ts (hunk 4 above)
Comment (ali-bahjati) [reply pyth-evm-price-pusher/src/evm-price-listener.ts:94]: Yeah.. I didn't know which is better. I will do what you suggest.

**Pull request merged**